### ISSUE TITLE
Design difference between Source Han Sans SC and Source Han Serif SC 2

### ISSUE THREAD

**Issue by dadasutasudadas** [adobe-fonts/source-han-serif#56]
21 characters were found in Source Han Serif SC that have different design with Source Han Sans SC.
 They are ⻁ (U+2EC1), ⻪ (U+2EEA), 懲 (U+61F2), 乑 (U+4E51), 堩 (U+5829), ⼬ (U+2F2C), 巆 (U+5DC6), 濹 (U+6FF9), 衋 (U+884B), 贗 (U+8D17), ⿋ (U+2FCB), 㗴 (U+35F4), 䙶 (U+4676), 䜛 (U+471B), 䞅 (U+4785), 䟡 (U+47E1), 𩻄 (U+29EC4), 㧤 (U+39E4), 㵮 (U+3D6E), 䎉 (U+4389), 䕎 (U+454E)
![170504_22glyphsquestion_1](https://cloud.githubusercontent.com/assets/21096794/25710139/549c2888-3126-11e7-80cc-30d8171bb7ec.JPG)
![170504_22glyphsquestion_2](https://cloud.githubusercontent.com/assets/21096794/25710145/5831ef64-3126-11e7-89d0-7d29eb66d7d3.JPG)
![170504_22glyphsquestion_3](https://cloud.githubusercontent.com/assets/21096794/25710194/81d73892-3126-11e7-96f3-853c3e58faaa.JPG)
![170504_22glyphsquestion_4](https://cloud.githubusercontent.com/assets/21096794/25710255/b31f2e8c-3126-11e7-832c-b363add92a24.JPG)

**Comment by kenlunde** [comment]
Thank you. A small number of these issues have been fixed for the forthcoming dot release, and some of them are _Source Han Sans_ issues, not _Source Han Serif_ ones. Also, some of them are intentional typeface design differences.

I will go through this list in the next few days to determine their dispositions.

**Comment by kenlunde** [comment]
Here's my analysis of the 21 glyph pairs that you identified:

⻁ U+2EC1	Serif → Fixed for Version 1.001
	Sans → OK
⻪ U+2EEA	Serif → Fixed for Version 1.001
	Sans → OK
懲 U+61F2	Serif → OK
	Sans → Known issue (fix CN glyph)
乑 U+4E51	Serif → OK
	Sans → Remove CN glyph; Use JP glyph for CN/TW
堩 U+5829	Serif → OK
	Sans → Known issue (fix CN glyph). See U+21377 𡍷
⼬ U+2F2C	Serif → Fixed for Version 1.001
	Sans → OK
巆 U+5DC6	Serif → Add CN glyph
	Sans → OK
濹 U+6FF9	Serif → OK
	Sans → Known issue (change mapping to JP variant)
衋 U+884B	Serif → OK
	Sans → Known issue (for both JP and CN glyphs)
贗 U+8D17	Serif → Rename TW glyph to be CN glyph
	Sans → OK
⿋ U+2FCB	Serif → OK
	Sans → CN glyph for U+9EF9 needs to be adjusted
㗴 U+35F4	Serif → OK
	Sans → Known issue (add CN glyph)
䙶 U+4676	Serif → Fix CN glyph
	Sans → OK
䜛 U+471B	Serif → Fix CN glyph
	Sans → OK
䞅 U+4785	Serif → Fix CN glyph
	Sans → OK
䟡 U+47E1	Serif → OK
	Sans → Fix CN glyph
𩻄 U+29EC4	Serif → OK (typeface design difference)
	Sans → OK (typeface design difference)
㧤 U+39E4	Serif → OK (typeface design difference)
	Sans → OK (typeface design difference)
㵮 U+3D6E	Serif → OK
	Sans → Remove CN glyph (it also has a subtle interpolation issue)
䎉 U+4389	Serif → OK
	Sans → Fix CN glyph 
䕎 U+454E	Serif → OK
	Sans → Known issue (fix CN glyph)

In other words, you found five _Source Han Serif_ issues (four new CN glyphs need to be added, and one TW glyph needs to be changed to a CN glyph) and five _Source Han Sans_ ones (two CN glyphs need to be removed, and three CN glyphs need to be fixed). The rest are either addressed in the forthcoming _Source Han Serif_ update, or are already known _Source Han Sans_ issues.

I will update the appropriate issues within the next day or two to consolidate the above, and then will close this issue.

**Comment by kenlunde** [comment]
(Thank you, @miguelsousa, for turning my data above into a easy-to-read table.)

The above changes are now either reflected in the appropriate issues (_Source Han Serif_) or in my own notes (_Source Han Sans_ Version 2.000), and therefore consolidated.
